docs/tutorial/typesfuns.rst

```diff
@@ -860,12 +860,12 @@ Dependent Pairs
 
 Dependent pairs allow the type of the second element of a pair to depend
 on the value of the first element. Traditionally, these are referred to
-as “sigma types”:
+as “sigma types”, so in older code you might see Sigma rather than DPair:
```

Comment:
Well, I was thinking of the beginner who encounters Sigma or MkSigma, and is puzzled. I'm familiar with Pi and Sigma types, but many newcomers to the language won't be. That's why I chose to add rather than remove stuff here.
I can't see much use of DPair compared with Sigma when i look at current code.